packages/components/src/menu-actions/Menu.tsx

```diff
@@ -132,20 +170,28 @@ class Menu extends PureComponent {
       e.key === 'ArrowUp' ||
       (e.key === 'Tab' && e.shiftKey === true)
     ) {
-      newFocus = ContextActionUtils.getNextMenuItem(newFocus, -1, menuItems);
+      newFocus = ContextActionUtils.getNextMenuItem(
+        newFocus || 0,
```

Comment:
Doesn't really matter here, but `??` instead of `||`
```suggestion
        newFocus ?? 0,
```